lib/compress.js

```diff
@@ -1051,7 +1051,7 @@ merge(Compressor.prototype, {
         });
         def(AST_Object, function(compressor){
             var val = {};
-            for (var i = this.properties.length; --i >= 0;) {
+            for (var i = 0, l = this.properties.length; i < l; i++) {
```

Comment:
This is a personal preference I admit, but I'd prefer `l` not be used as a variable name because it looks like `1`. Could you please use `len` instead?